geode-redis/src/main/java/org/apache/geode/redis/internal/executor/CommandFunction.java

```diff
@@ -117,6 +118,32 @@ protected Object compute(Region localRegion, ByteArrayWrapper key,
             () -> new RedisSetInRegion(localRegion).sscan(key, matchPattern, count, cursor);
         break;
       }
+      case SCREATE: {
```

Comment:
How is this used / invoked?

Reply:
I think the place I was using it was refactored and now it is not used. I'll remove it.